docs/content/developer/cryptography/transaction-auth/offline-signing.mdx

```diff
@@ -82,15 +82,15 @@ Mutated Objects:
 Alternatively, you can use the active key in IOTA Keystore to sign and output a Base64-encoded sender signed data with flag `--serialize-signed-transaction`. 
 
 ```shell
-iota client transfer-iota --to <IOTA-ADDRESS> --iota-coin-object-id <COIN-OBJECT-ID> --serialize-signed-transaction
+iota client transfer --to <IOTA-ADDRESS> --object-id <COIN-OBJECT-ID> --serialize-signed-transaction
```

Comment:
Here too?

Reply:
Changed in https://github.com/iotaledger/iota/pull/4701/commits/75a87379841387a37fb081e47d782fff88c49f98